docs/sources/flow/_index.md

```diff
@@ -11,6 +11,12 @@ Grafana Agent Flow is a _component-based_ experimental revision of Grafana
 Agent with a focus on ease-of-use, debuggability, and ability to adapt to the
 needs of power users.
 
+Components allow for reusability, composability and focus on a single task. 
+
+* **Reusability** allows for components to have multiple inputs and outputs.
```

Comment:
Reusability feels off but not sure how else to describe it.

Reply:
I don't understand how having multiple inputs and outputs to a component would make it reusable.

Reply:
I think of reusable as a black box that you can snap in somewhere else.

Reply:
What about this?

> **Reusability** allows for the output of components to be reused in the input for multiple other components.

Reply:
IMO the intent is that unlike our static agent where something like the discovery of targets cant be shared, in this it can.

Reply:
@rfratto, your description sounds better than mine!